scripts/release/publish-commands/publish-to-npm.js

```diff
@@ -8,10 +8,21 @@ const {join} = require('path');
 const semver = require('semver');
 const {execRead, execUnlessDry, logPromise} = require('../utils');
 
-const push = async ({cwd, dry, packages, version}) => {
+const push = async ({cwd, dry, packages, version, tag}) => {
   const errors = [];
-  const isPrerelease = semver.prerelease(version);
-  const tag = isPrerelease ? 'next' : 'latest';
+  let isPrerelease;
```

Comment:
I _think_ `isPrerelease` should always be determined by:
```js
isPrerelease = semver.prerelease(version);
```

And I think the script should fail (with an error) if we ever try to combine the "latest" tag with a version that isn't x.x.x (eg 16.3.0-alpha.0 and "latest" should be an error)